src/BuildScriptGenerator/Node/NodeBashBuildSnippetProperties.cs

```diff
@@ -19,7 +19,8 @@ public class NodeBashBuildSnippetProperties
             string compressNodeModulesCommand,
             string compressedNodeModulesFileName,
             bool configureYarnCache = false,
-            bool pruneDevDependencies = false)
+            bool pruneDevDependencies = false,
+            string oryxAppInsightInjectCommand = null)
```

Comment:
why the `oryx` prefix? we didn't add it to other param names (e.g. `compressNodeModulesCommand`)